app/models/competition.rb

```diff
@@ -1251,24 +1237,20 @@ def colliding_registration_start_competitions
   end
 
   # Source http://www.movable-type.co.uk/scripts/latlong.html
-  def kilometers_to(c)
+  def kilometers_to(competition)
     6371 *
       Math.sqrt(
-        (((c.longitude_radians - longitude_radians) * Math.cos((c.latitude_radians + latitude_radians)/2)) ** 2) +
-        ((c.latitude_radians - latitude_radians) ** 2),
+        (((competition.longitude_radians - longitude_radians) * Math.cos((competition.latitude_radians + latitude_radians)/2)) ** 2) +
+        ((competition.latitude_radians - latitude_radians) ** 2),
       )
   end
 
-  def has_date?
+  def date_set?
```

Comment:
Isn't this the exact same as the old `has_defined_dates?` (at the current state of in-progress review: `dates_set?`) above?!

Reply:
yeah, looks like this is a duplicate